gpt_researcher/retrievers/searx/searx.py

```
@@ -73,6 +73,6 @@ def search(self, max_results: int = 10) -> List[Dict[str, str]]:
             return search_response
 
         except requests.exceptions.RequestException as e:
-            raise Exception(f"Error querying SearxNG: {str(e)}")
+            raise Exception(f"Error querying SearxNG: {e!s}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)